controllers/apps/transformer_cluster_credential.go

```diff
@@ -47,9 +49,23 @@ func (c *ClusterCredentialTransformer) Transform(ctx graph.TransformContext, dag
 		return err
```

Comment:
perhaps refactor this transformer to `ClusterConnCredTransformer`, `CredentialTransformer` is inaccurate